### ISSUE TITLE
Collections icon

### ISSUE THREAD

**Issue by nathansherburn** [SLC3/MARS#94]
Just thinking from a presenter point of view - a **collection** is all based around the group of people you're presenting to over the course of a semester, year, conference, workshop etc.

In other words, collections are all about the people subscribed to the feed.

Would it make sense then to have the icon for collections as a group of people? Essentially a collection of _people_ rather than questions - especially since questions may be identical across some collections.

What do you guys think? Would treating **collections** as **collections of people** make sense throughout the UI (icons etc)

![groups](https://cloud.githubusercontent.com/assets/3915868/16339051/d881a746-3a63-11e6-89ef-36f86a21fba3.png)
?

**Comment by nathansherburn** [comment]
@thanatos6666 I remember that, based on Marc's UX advice, we were also looking to change the text from collection to "Unit".

I feel like "Audience" might be a better name given that collections are more strictly defined by the people subscribed rather than the unit they belong to. This also removes the need to make it configurable (ie let users set it to "unit", "subject" etc)

**Comment by nathansherburn** [comment]
We decided that "collections" are a better way to think about this concept than "audiences" or "units" as cloning a collection clones the questions and archiving archives the questions (with their results). It is hopefully more straight forward than audiences for presenters.

We will make a dictionary that allows us to change UI words in the future though.

I've also changed the icon from people to folders as it's more appropriate given this definition.
